packages/perseus/src/components/__stories__/image-loader.stories.tsx

```diff
@@ -1,60 +1,40 @@
-/* eslint-disable @khanacademy/ts-no-error-suppressions */
 import * as React from "react";
 
-type StoryArgs = Record<any, any>;
-
-type Story = {
-    title: string;
-};
-
 import ImageLoader from "../image-loader";
 
+import type {Meta, StoryObj} from "@storybook/react";
+
 const svgUrl = "http://www.khanacademy.org/images/ohnoes-concerned.svg";
 const imgUrl = "https://www.khanacademy.org/images/hand-tree.new.png";
 
-export default {
+const meta: Meta = {
     title: "Perseus/Components/Image Loader",
-} as Story;
-
-export const SvgImage = (args: StoryArgs): React.ReactElement => {
-    return (
-        <ImageLoader
-            src={svgUrl}
-            preloader={null}
-            imgProps={{
-                alt: "ALT",
-            }}
-            onUpdate={() => {}}
-        />
-    );
+    component: ImageLoader,
+    args: {
+        preloader: null,
+        imgProps: {
+            alt: "ALT",
+        },
+        onUpdate: () => {},
+    },
+};
+export default meta;
+
+type Story = StoryObj<typeof ImageLoader>;
+
+export const SvgImage: Story = {
+    args: {
+        src: svgUrl,
+    },
```

Comment:
I'd disable chromatic on this, since I don't think the snapshot is working as intended.

```suggestion
    },
    
    parameters: {
        chromatic: {
            // Disabling because this isn't correctly testing the SVG, just a loading a spinner.
            disableSnapshot: true,
        },
    }
```

<img width="1311" alt="image" src="https://github.com/user-attachments/assets/98ac38fc-d6c2-4177-98d6-7d0ce439923e">

Reply:
Good call!  I've disabled this image-loader for snapshots, but added a delay to `SvgImage` as I think it's useful to see snapshots for that component.